test/webpack-build.test.js

```diff
@@ -193,18 +197,57 @@ test('Bundles can use custom names', async (test) => {
   test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda', 'service.js')));
 });
 
-test('Multi-entry bundling cannot be used with bundle zipping', async (test) => {
-  const options = {
+test('Bundles for multiple entries can be zipped', async (test) => {
+  await build({
     entrypoint: [
-      path.join(__dirname, 'fixtures', 'lambda_service.js'),
-      path.join(__dirname, 'fixtures', 'lambda_graphql.js')
+      path.join(__dirname, 'fixtures', 'lambda_graphql.js:graphql.js'),
+      path.join(__dirname, 'fixtures', 'lambda_service.js:lambda/service.js')
     ],
     outputPath: test.context.buildDirectory,
     serviceName: 'test-service',
     zip: true
-  };
+  });
+
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'graphql.js.zip')));
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda/service.js.zip')));
+});
+
+test.serial('Expand input entrypoint directory into multiple entrypoints', async (test) => {
```

Comment:
Marking this as `serial` only causes it to be serialized wrt the other test cases in this file. This file will still be run in parallel with the other test files. This means that even though the directory name is unique and gets cleaned up, it still represents shared test state.

Reply:
That's ok. There's a separate process for each test file so this should avoid interference caused by replacing `fs.stat` with a stub.

Reply:
^ just to clarify, I added `test.serial` due to replacing `fs.stat` to simulate read error while searching for index file. I didn't want this temporary `fs.stat` replacement to interfere with other files.

Reply:
Sorry, I should be more clear. This is the test case with the `empty` directory. My comment is in reference to that. I agree that `fs.stat()` should be fine.